core/src/components/modal/gestures/sheet.ts

```diff
@@ -110,6 +116,33 @@ export const createSheetGesture = (
     baseEl.classList.add(FOCUS_TRAP_DISABLE_CLASS);
   };
 
+
+  /**
+   * Used when `scrollAtEdge` is disabled.
+   * Changes the footer that is currently visible
+   * @param footer - The footer to show
+   */
+  const swapFooterVisibility = (footer: 'original' | 'cloned') => {
+    const footerToHide = footer === 'original' ?
+      wrapperEl.nextElementSibling as HTMLIonFooterElement :
+      baseEl.querySelector('ion-footer') as HTMLIonFooterElement;
+    const footerToShow = footer === 'original' ?
+      baseEl.querySelector('ion-footer') as HTMLIonFooterElement :
+      wrapperEl.nextElementSibling as HTMLIonFooterElement;
+
+    footerToShow.style.removeProperty('display');
+    footerToShow.removeAttribute('aria-hidden');
+
+    const pagePadding =
+      footer === 'original' ? 0 : footerToShow.clientHeight;
+
+    const page = baseEl.querySelector('.ion-page') as HTMLElement;
+    page.style.setProperty('padding-bottom', `${pagePadding}px`);
```

Comment:
```suggestion
    if (footer === 'original') {
      page.style.removeProperty('padding-bottom');
    } else {
      const pagePadding = footerToShow.clientHeight;
      page.style.setProperty('padding-bottom', `${pagePadding}px`);
    }
```

Since we added the inline padding bottom, then we can safely remove it so it reverts to the original inline styles.